docs/specs/oidc.md

```diff
@@ -461,44 +470,114 @@ Silent Authentication
 1. If authorization code request result is successful, send the token request to Authgear with code + code verifier
 1. Authgear returns token response to SDK with id token + new access token
 
-## First-party Clients
+## Clients
+
+The value of `x_application_type` determines the type of the client.
 
-OAuth protocol allows user to delegate access to clients, so that clients can
-act on behalf of the user.
+The tables summarizes client Party, confidentiality, PII in ID token, and access to privileged user operations of different `x_application_type`.
 
-Usually, OAuth clients are third-party clients, with limited trust:
-- User need to give consent to a client before clients is allowed access.
-- Privileged user operations (e.g. change password) is usually not exposed
-  through APIs accessible through OAuth protocol.
+|`x_application_type`|Client Party|Confidentiality|PII in ID token|Access to privileged user operations|
+|---|---|---|---|---|
+|`spa`|First-party|public|No|Yes|
+|`traditional_webapp`|First-party|public|No|Yes|
+|`native`|First-party|public|No|Yes|
+|`confidential`|First-party|confidential|Yes|No|
+|`third_party_app`|Third-party|confidential|Yes|No|
```

Comment:
I think `third_party_app` is supposed to be legacy, so we may mark it as such.

Reply:
In the future we may bring it back. So I did not mark it as legacy. In the implementation, I will "hide" it.